src/pixbuf.rs

```diff
@@ -194,7 +194,15 @@ impl Pixbuf {
         })
     }
 
-    pub unsafe fn get_pixels(&self) -> &mut [u8] {
+    pub fn get_pixels(&self) -> &[u8] {
```

Comment:
This is fine

Reply:
Actually not: https://github.com/gtk-rs/gdk-pixbuf/pull/77#issuecomment-393543649